libs/category/src/models/filters/type/range/request-base.ts

```diff
@@ -0,0 +1,8 @@
+import { DaffCategoryFilterRangeRequestOption } from '../range/public_api';
+import { DaffCategoryFilterRangeType } from './filter-type';
+
+export interface DaffCategoryFilterRangeRequestBase<T> {
+	type: DaffCategoryFilterRangeType;
+	name: string;
+	value: DaffCategoryFilterRangeRequestOption<T>;
```

Comment:
Is it intentional that you can only add one ranged filter to a request at a time?